shared/utils/replaceAbbreviations.js

```diff
@@ -10,8 +10,10 @@ const abbreviations = {
     'E': 'English',
     'enc': 'enclitic',
     'esp': 'especially',
-    'ext. stuff': 'inflectional suffix',
-    'int': 'interjection',
+    'ext. suff': 'extensional suffix',
+    'H': 'Hausa',
+    'infl. suff': 'inflectional suffix',
+    'int': 'interjection/interrogative',
```

Comment:
Since `int.` represents both interject and interrogative, I'm including them both for the same abbreviation.